src/input/readAndValidateInput.ts

```diff
@@ -108,7 +113,7 @@ export const getChoice = async (
     choice !== null &&
     choice !== undefined
   ) {
-    return choice + 1
+    return choice
```

Comment:
This had to change because `prompts` now returns the `value` which has been made mandatory.